### PR TITLE
Feature 71 syntehtic testing data

### PR DESCRIPTION
**Pull request by andrewelamb**
fixes #73

### CODE REVIEW

File: R/data_example_functions.R [hunk 1]
```diff
@@ -1,149 +1,62 @@
...
-  "nf_publication_status_module" %>%
+get_new_submissions_config <- function(){
+  "new_submissions_module" %>%
     get_json_path() %>%
     jsonlite::read_json()
 }
 
-get_nf_new_submissions_config <- function(){
-  "nf_new_submissions_module" %>%
+get_nf_synapse_config <- function(){
```
Comment (milen-sage) [inline R/data_example_functions.R:52]: @andrewelamb should this one be nf-specific?

Comment (milen-sage) [reply R/data_example_functions.R:52]: nm, I think I see

**Review by milen-sage — APPROVED** [review]
The overall structure of data looks good! W/ v2 we can revisit some of the modules' and synthetic data features to simplify and align with our data flow. 

Is there nothing in tests/testthat/helpers.R?

**Comment by andrewelamb** [comment]
@milen-sage Yeah my thoughts were I was going to add something to helpers.R, but then never needed to and didn't delete it.

**Pull request merged**